retrieval/reports/archie_toolcall_probe.py

```
@@ -0,0 +1,61 @@
+CASES = [("json-schema", SYS_JSON), ("xml-tag", SYS_XML), ("minimal", SYS_MIN)]
+
+def parseable(text):
+    """Did it emit something a dispatcher could actually execute?"""
+    hits = []
+    try:
+        m = re.search(r'\{[^{}]*"tool"\s*:\s*"web_search"[^{}]*\{[^{}]*\}[^{}]*\}', text, re.S)
+        if m: json.loads(m.group(0)); hits.append("json")
+    except Exception: pass
+    if re.search(r'<tool_call>\s*web_search\s*\(', text): hits.append("xml")
+    if re.search(r'CALL\s+web_search\s*\(\s*["\']', text): hits.append("minimal")
+    if re.search(r'\bweb_search\b', text): hits.append("mentions-name")
```

Ruff (0.16.3)

[error] Multiple statements on one line (colon)

(E701)